source/UberRides/RideRequestButton.swift

```diff
@@ -105,25 +105,31 @@ import CoreLocation
         sizeToFit()
     }
     
-    // add title, image, and sizing configuration
     override func setContent() {
         super.setContent()
         
+        configureTitleLabel()
+        configureImage()
+    }
+    
+    private func setSizesConfigs() {
         uberTitleLabel.font = UIFont.systemFontOfSize(17)
         uberTitleLabel.numberOfLines = 2;
         uberTitleLabel.textAlignment = .Right
-        
-        // Add title label
+    }
+    
+    private func configureTitleLabel() {
+        setSizesConfigs()
```

Comment:
Again, I think `setSizesConfigs()` is a little overkill. Can we just pull the 3 lines from that method into `configureTitleLabel()` ?